src/test/java/dev/ocpd/jsensible/rules/spring/NoNullableMisuseInRepositories.java

```diff
@@ -1,102 +1,31 @@
 package dev.ocpd.jsensible.rules.spring;
 
-import jakarta.persistence.Entity;
-import jakarta.persistence.Id;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 @SuppressWarnings("unused")
 public class NoNullableMisuseInRepositories {
 
-    @Entity
-    public static class EntityCompliant {
-
-        @Id
-        private Long id;
-
-        private String test1;
-
-        private String test2;
-
-        public Long getId() {
-            return id;
-        }
-
-        public String getTest1() {
-            return test1;
-        }
-
-        public String getTest2() {
-            return test2;
-        }
-    }
-
-    public interface CompliantRepo extends JpaRepository<EntityCompliant, Long> {
+    public interface CompliantRepo extends JpaRepository<Object, Long> {
```

Comment:
```suggestion
    public interface Compliant extends JpaRepository<Object, Long> {
```
Let's be consistent with our namings